deploy/test/test_server_config.rb

```diff
@@ -95,10 +95,16 @@
           })
 
         @s.bootstrap.must_equal true
+#        sleep(10)
         @s.validate_install.must_equal true
       end
     end
 
+    it "should deploy triggers" do
+      @s.deploy_triggers.must_equal true
+      @s.clean_triggers.must_equal true
+    end
+
```

Comment:
This ran for me because I already had a triggers-config file in deploy. Just revised so that it used the one in the sample directory for the test.